spec/spec_helper.rb

```diff
@@ -294,3 +294,9 @@ def self.new(*args)
     MockIMAP.new
   end
 end
+
+module Kernel
+  def warn(m)
+    $stderr.puts(m)
+  end
+end
```

Comment:
@jeremyevans Why is Kernel#warn redefined here? Shouldn't it already exist anyway?

Notably this defining is incompatible with multi-arguments `warn`.
As a result it causes https://github.com/oracle/truffleruby/issues/1900

Reply:
@eregon Not sure why it was added.  It should be safe to remove if it doesn't break any specs.